openlibrary/templates/books/custom_carousel.html

```diff
@@ -30,11 +30,7 @@
     $ author_names = ''
   $ modifier = ''
   $ work_key = book.key if book.key.startswith('/work') else book.get('work_key')
-
-  $if lazy:
-    $ img_attr = 'data-lazy'
-  $else:
-    $ img_attr = 'src'
+  $ img_attr = 'src'
```

Comment:
Ah, `data-lazy` is a special attr for slick carousels, so we do still want it, unfortunately

Reply:
Cool.  Where is the code that uses this attribute?

Reply:
It's inside slick; see its docs: https://kenwheeler.github.io/slick/